src/components/atoms/chipsForEnum/ChipsForEnum.tsx

```diff
@@ -1,23 +1,36 @@
-import { Chip } from '@mui/material'
-import { Genre, Role } from '../../../common/constants/enums'
+import { Chip, useTheme } from '@mui/material'
+import { Link, useNavigate } from 'react-router-dom'
 import { enumMappings } from '../../../common/constants/constants'
-import { useNavigate } from 'react-router-dom'
+import { Genre, Role } from '../../../common/constants/enums'
 
 export function ChipsForEnum({ value }: { value: Genre | Role }): JSX.Element {
   const { label, url } = enumMappings[value]
   const navigate = useNavigate()
+  const theme = useTheme()
+  const buttonColor = theme.palette[value].main
 
   const handleClick = () => {
     navigate(url)
   }
 
   return (
     <Chip
+      component={Link}
```

Comment:
Swtich to using explicit `Link` component

Reply:
Very cool